Dockerfile

```diff
@@ -1,15 +1,26 @@
+FROM node:16 AS builder
+
+WORKDIR /app
+
+# Install the project's dependencies, including dev dependencies.
+COPY package.json yarn.lock ./
+RUN yarn install --pure-lockfile --production=false
+
 FROM node:16
```

Comment:
As above we can probably use the `-slim` image.

Reply:
Done.